### PR TITLE
Psec 1483 - WIP

### PR DESCRIPTION
**Pull request by Kabbas99**
Multiple files added that create some basic account and title filters, along with security and some testing

### CODE REVIEW

File: auditing/logfile.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package auditing
+
+import (
+    "bytes"
+    "log"
+    "os"
+)
+// Log logs information using the standard Go logger package
+func Log(logMessage string) {
```
Comment (cob16) [inline auditing/logfile.go:9]: A future improvement might be to use a premade logging framework such as https://github.com/sirupsen/logrus

File: cmd/InspectorManager/main.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	if err != nil {
+		auditing.Log(err.Error())
+		os.Exit(1)
+	}
+	stsServiceFactory := clients.NewSTSClientSessionConfig()
+    err = security.AssumeAccountRole(&myUserInput, stsServiceFactory, myUserInput.AwsAccount)
+	if err!= nil {
+		auditing.Log(err.Error())
+		os.Exit(1)
+	}
```
Comment (cob16) [inline cmd/InspectorManager/main.go]: This assumes a certain local config that sadly will not work for half the team. In other projects we have offloaded role management outside of the project and simply inject the correct credentials using a tool such as `aws-profile` or `aws-vault`

**Review by mteasdal2 — DISMISSED** [review]

File: inspector/inspector2.go [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            titleFilters = append(titleFilters, accountFilter)
+        }
+    }
+    i.CVETitleFilters = titleFilters
+    return i
+}
+
+func (i *InspectorFilterPipeline) PopulateAccountFilters(comparisonOperator string) *InspectorFilterPipeline {
+    var accountFilters []types.StringFilter
+    if len(i.AWSAccounts) > 0 {
```
Comment (cob16) [inline inspector/inspector2.go]: Could we get rid of this if statment? I had assumed that if AWSAccounts was empty the range cmd would do nothing :shrug:

File: security/security.go [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+			RoleSessionName: &userCredentials.SessionName,
+		})
+	if err != nil {
+		return err
+	}
+	userCredentials.ServiceCredentials.AccessKeyId = *assumeRoleResult.Credentials.AccessKeyId
+	userCredentials.ServiceCredentials.SecretAccessKeyId = *assumeRoleResult.Credentials.SecretAccessKey
+	userCredentials.ServiceCredentials.SessionToken = *assumeRoleResult.Credentials.SessionToken
+	return nil
+}
```
Comment (cob16) [inline security/security.go:44]: all of this can be forgone if we push credential management outside of the program. (see last comment for reason)

**Review by cob16 — DISMISSED** [review]

**Review by ulaskan — APPROVED** [review]

**Pull request merged**